mem-03-D.xml

```diff
@@ -10742,9 +10742,9 @@ The logo of Star Alliance, the largest airline alliance on Earth, is often paint
       <column name="definition_zh_HK">中午</column>
       <column name="definition_pt">meio dia</column>
       <column name="definition_fi">keskipäivä</column>
-      <column name="synonyms"></column>
+      <column name="synonyms">{pemjep:n}</column>
```

Comment:
So there was actually a reason why {DungluQ} and {pemjep} were not already synonyms. People weren't sure they were quite the same thing, as we don't know exactly how time-keeping works on {Qo'noS}. It's also not clear whether one or both of these refer to just a point in time or a period of time (as "midday" can sometimes refer to a period of time around noon in English). But since these are both already marked as antonyms of {ramjep}, making them synonyms seems like the right thing.